app/src/main/java/com/github/h3lp3rs/h3lp/notification/EmergencyListener.kt

```diff
@@ -1,66 +1,112 @@
 package com.github.h3lp3rs.h3lp.notification
 
+import LocationHelper
 import android.content.Intent
 import android.os.Bundle
 import com.github.h3lp3rs.h3lp.*
 import com.github.h3lp3rs.h3lp.database.Databases
+import com.github.h3lp3rs.h3lp.database.Databases.*
 import com.github.h3lp3rs.h3lp.database.Databases.Companion.databaseOf
 import com.github.h3lp3rs.h3lp.dataclasses.EmergencyInformation
 import com.github.h3lp3rs.h3lp.dataclasses.HelperSkills
+import com.github.h3lp3rs.h3lp.locationmanager.GeneralLocationManager
+import com.github.h3lp3rs.h3lp.notification.NotificationService.Companion.createNotificationChannel
+import com.github.h3lp3rs.h3lp.notification.NotificationService.Companion.sendIntentNotification
 import com.github.h3lp3rs.h3lp.signin.SignInActivity
+import com.github.h3lp3rs.h3lp.signin.SignInActivity.Companion.globalContext
 import com.github.h3lp3rs.h3lp.storage.Storages
+import com.github.h3lp3rs.h3lp.storage.Storages.*
+import com.github.h3lp3rs.h3lp.storage.Storages.Companion.storageOf
 
 object EmergencyListener {
 
+    // Maximum helpee-helper distance in meters
+    private const val MAX_DISTANCE = 5000
+
     /**
      * Activates all listeners for helpees the helper may help
      */
     fun activateListeners() {
-        val skillStorage = Storages.storageOf(Storages.SKILLS)
-        val skills = skillStorage.getObjectOrDefault(SignInActivity.globalContext.getString(R.string.my_skills_key), HelperSkills::class.java, null)
-        if(skills == null) return
+        val skillStorage = storageOf(SKILLS)
+        val skills = skillStorage.getObjectOrDefault(
+            SignInActivity.globalContext.getString(R.string.my_skills_key),
+            HelperSkills::class.java,
+            null
+        )
+        if (skills == null) return
         else {
```

Comment:
```suggestion
         skills?.let {
```
This is just a suggestion ;)